src/Illuminate/Cache/FileStore.php

```diff
@@ -35,6 +35,11 @@ class FileStore implements Store, LockProvider
      */
     protected $filePermission;
 
+    /**
+     * The path to the lock directory.
+     */
+    protected ?string $lockPath = null;
```

Comment:
Please remove the property typehint here and add a `@var` annotation

```suggestion
     *
     * @var string|null $lockPath
     */
    protected $lockPath;
```